packages/docs/stories/components/button.stories.ts

```diff
@@ -16,12 +10,23 @@ const { args, argTypes } = storybookDefaults('syn-button');
 const { overrideArgs } = storybookHelpers('syn-button');
 const { generateTemplate } = storybookTemplate('syn-button');
 
+const generateStoryParameters = (attributeName: string) => {
```

Comment:
Please reduce this function on only the description, not the rest of the parameters. :)